lib/std/Target.zig

```diff
@@ -1205,7 +1205,7 @@ pub const Cpu = struct {
         pub const Set = struct {
             ints: [usize_count]usize,
 
-            pub const needed_bit_count = 288;
+            pub const needed_bit_count = 398;
```

Comment:
oof.

Just out of curiosity, has there been any investigation into whether *all* of these SPIR-V extensions are actually even useful to know about for a compiler (or compiler-adjacent tooling)?

Reply:
Well, Compiler clearly has no idea about %95 of these and usually don't have to but since features are currently the only way to declare capabilities/extensions we need to keep them for now. I think we can get rid of those that compiler doesn't care about by moving the capability/extension emitting logic from linker to codegen since we have a deduplication pass already